Assets/Scripts/CrosshairController.cs

```diff
@@ -6,8 +6,11 @@ public class CrosshairController : MonoBehaviour
 
     void Start()
     {
-            //Hide cursor for Production Builds 
-            Cursor.visible = Debug.isDebugBuild;
+            #if DEBUG
```

Comment:
does unity has a way to determine when something is production build?

Reply:
no unity do not have this option in the preprocessor. Fixed to unity method